Source/WebCore/dom/SpatialBackdropSource.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2025 Apple Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#if ENABLE(WEB_PAGE_SPATIAL_BACKDROP)
+#pragma once
+
+#include <wtf/URL.h>
+
+namespace WebCore {
+
+struct SpatialBackdropSource {
+public:
+    WEBCORE_EXPORT URL sourceURL;
+    WEBCORE_EXPORT URL modelURL;
+    WEBCORE_EXPORT std::optional<URL> environmentMapURL;
+
+    WEBCORE_EXPORT SpatialBackdropSource(const URL&, const URL&, const std::optional<URL>);
+    SpatialBackdropSource(const URL& sourceURL, const URL& modelURL, const URL& environmentMapURL);
```

Comment:
I think we can use r-value references for all parameters here and WTFMove() internally for the initialization.